src/DGtal/dec/doc/packageDEC.dox

```diff
@@ -60,14 +62,58 @@ Basic operators, such as Hodge duality operator or exterior derivative, can be c
 @b Related @b documentation @b pages
 
 @b Related @b examples
-- testDiscreteExteriorCalculus.cpp
+- exampleDECSurface.cpp
+- examplePropagation.cpp
 - exampleDiscreteExteriorCalculusUsage.cpp
 - exampleDiscreteExteriorCalculusSolve.cpp
-- examplePropagation.cpp
+- exampleDiscreteExteriorCalculusChladni.cpp
+- testDiscreteExteriorCalculus.cpp
 - testLinearStructure.cpp
 - testEmbedding.cpp
-- exampleDECSurface.cpp
 
 */
 
+
+
+/**
+   Example of Discrete Exterior Calculus with Chladni.
```

Comment:
As an example to my comment: \example and @example must be moved to the example source code.